setup/gcp/scripts/run_forseti.sh

```diff
@@ -36,12 +36,33 @@ sudo systemctl restart forseti.service
 # Wait until the service is started
 sleep 10s
 
+# Set the output format to json
+forseti config format json
+
 # Run inventory command
 MODEL_NAME=$(/bin/date -u +%Y%m%dT%H%M%S)
 echo "Running Forseti inventory."
 forseti inventory create --import_as ${MODEL_NAME}
```

Comment:
Can you please add a TODO here, linking it to issue #1296?  Ultimately, we don't want to keep polling.

Reply:
Updated.